### DISCUSSION TITLE
Community Meeting 2023-07-12

### DISCUSSION THREAD

**Discussion by JamieMagee** [microsoft/component-detection#607] [Announcements]
📆 Wednesday 12th July 2023
🕙 10AM Pacific Time ([Find in your timezone](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20230712T17))
☎️ [Teams meeting](https://teams.microsoft.com/l/meetup-join/19%3ameeting_YmVmMjcxMmEtMmYwYi00MGNiLTg4MDgtODNiMDY5ZjNkYjY3%40thread.v2/0?context=%7b%22Tid%22%3a%2272f988bf-86f1-41af-91ab-2d7cd011db47%22%2c%22Oid%22%3a%22bb41e436-5c77-44e2-b783-1ad2b9f7aa7f%22%7d)

## Agenda

Comment agenda items below

- [Issues opened since last meeting](https://github.com/microsoft/component-detection/issues?q=is%3Aissue+created%3A%3E2023-06-14)
- [PRs opened since last meeting](https://github.com/microsoft/component-detection/pulls?q=is%3Apr+created%3A%3E2023-06-14)
- Discussion of issues:
  - #652
  - #653
  - #589

## Attendees
- @cobya
- @melotic
- @grvillic
- @alimohamad
- @amitla1 

## Recording

https://github.com/microsoft/component-detection/assets/1358764/bdec13ec-128f-4b88-984b-00a96af624ea

**Comment by cobya** [comment]
Updated agenda and attendees list post-meeting. Recording is being processed.
